deploying-daml/daml-on-fabric/fabric_run_network.md

```diff
@@ -0,0 +1,9 @@
+Alright time to start up Fabric. The git repo we're cloning has an example configuration for running a few peers and is a great starting point if you decide to do your own deployment. If you go that route the DAML-on-Fabric [README](https://github.com/digital-asset/daml-on-fabric/blob/master/README.md) and [DEPLOYMENT GUIDE](https://github.com/digital-asset/daml-on-fabric/blob/master/DEPLOYMENT_GUIDE.md) are must-reads.
+
```

Comment:
The process is done when you see the `peer0.org2.example.com_1  | 2020-06-16 17:18:36.639 UTC [fsblkstorage] preResetHtFiles -> INFO 01d No active channels passed` message in the terminal.